src/payment/data/sagas.js

```diff
@@ -34,6 +37,10 @@ function* isBasketProcessing() {
   return yield select(state => state.payment.basket.isBasketProcessing);
 }
 
+function* isCaptureKeyProcessing() {
+  return yield select(state => state.payment.isCaptureKeyProcessing);
```

Comment:
Given `combineReducers` above, I'm surprised this doesn't need to be `state.payment.captureKey.isCaptureKeyProcessing`?

Reply:
I think you're probably right; I'll experiment some in the morning.  I believe this bug doesn't come up in normal operation, but would break the "don't start another request if one is in-flight" guard.